meetings/2021-11-01.md

```diff
@@ -0,0 +1,124 @@
+
+ACTION: acoburn will follow up in issue about non-public WebID Profile documents
+
+ACTION: pavlik will follow up in PR about implicit trust
+
+
+
+### Other specs and requirements on Client ID Document
+
+https://solid.github.io/solid-oidc/#clientids-document
+> When a Client Identifier is dereferenced, the resource MUST be serialized as an `application/ld+json` document unless content negotiation requires a different outcome.
+
+
+https://github.com/solid/data-interoperability-panel/issues/210#issuecomment-953401259
+> This format has two advantages -- first, it allows apps to host static JSON-LD documents as part of their deployment. Many Solid apps are composed of static JavaScript, so this doesn't add a requirement for a content-negotiable backend.
+
+* Pavlik: at present, the client id document MUST be serializable as JSON-LD via content negotiation. This implies that the server is required to support that. But that seems to contradict the possiblity of a static document.
+* Aaron: Let's take example of server which doesn't do any content negotiation; it either ignores it or returns 406, which follows the HTTP spec. If the server supports content negotiation, it would only return those representations that it supports.
+* Pavlik: we don't want to require Turtle to support Turtle variant? We currently have some work standardizing application WebID document. We should be careful in coordinating with other specifications. That may complicate other specs that have other requirements. They may expect Turtle; otherwise, they would need to support JSON-LD, too. This affects at least the interop spec.
+* Aaron: It goes to the question of what the Client ID Document is. For purposes of Solid-OIDC, it's just a structure to support login. We could overload it to be an Application Profile. I would suggest that this would be a different resource.
+* Pavlik: but if we are denoting an application by an IRI, if we want to use the same IRI to denote the application across specifications, it's not so easy to say that there should just be a different document for the profile.
+* Aaron: Solid protocol requires both Turtle and JSON-LD for resource. Client should be able to expect one or the other. I would like to know what are the specific limitations when those documents are not content negotiable into turtle.
+* Pavlik: protocol requires at least Turtle and JSON-LD. That allows a client to have either Turtle or JSON-LD.
+* Aaron: We are coming from two different directions. Is `client_id` document in the storage or not? How you answer this question leads you in different directions. It's possible to have those identifiers hosted independently from app. One could have app registry where you would find app identifiers; there one would find `redicrect_uri`, etc. This could lead to app registries, etc.
```

Comment:
```suggestion
* Aaron: We are coming from two different directions. Is `client_id` document in the storage or not? How you answer this question leads you in different directions. It's possible to have those identifiers hosted independently from app. One could have app registry where you would find app identifiers; there one would find `redirect_uri`, etc. This could lead to app registries, etc.
```